examples/specify_updates.py

```diff
@@ -0,0 +1,72 @@
+from aiogram.types.inline_keyboard_button import InlineKeyboardButton
+from aiogram.types.inline_keyboard_markup import InlineKeyboardMarkup
+from aiogram.dispatcher.router import Router
+from aiogram.utils.handlers_in_use import get_handlers_in_use
+import logging
+
+from aiogram import Bot, Dispatcher
+from aiogram.types import Message, ChatMemberUpdated, CallbackQuery
+
+TOKEN = "6wo"
+dp = Dispatcher()
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+@dp.message(commands={"start"})
+async def command_start_handler(message: Message) -> None:
+    """
+    This handler receive messages with `/start` command
+    """
+
+    await message.answer(f"Hello, <b>{message.from_user.full_name}!</b>", reply_markup=InlineKeyboardMarkup(inline_keyboard=[[InlineKeyboardButton(text="Tap me, bro", callback_data="*")]]))
```

Comment:
Why this line is too long? (189 symbols)
Lines shouldn't be longer than 99 symbols.